src/Microsoft.Identity.Client/IByRefreshToken.cs

```diff
@@ -52,11 +52,16 @@ public interface IByRefreshToken
         Task<AuthenticationResult> AcquireTokenByRefreshTokenAsync(IEnumerable<string> scopes, string refreshToken);
 
         /// <summary>
-        /// TODO(migration): add documentation
+        /// Acquires an access token from an existing refresh token and stores it and the refresh token into 
+        /// the application user token cache, where it will be available for further AcquireTokenSilentAsync calls.
```

Comment:
>application user token cache [](start = 16, length = 28)

is this for conf client or public client? or both? asking because cc has user and app cache, so this is a little confusing on which token is getting cached where.